support/metrics/metricsTracker.go

```diff
@@ -81,6 +86,8 @@ type commonProps struct {
 	OperationalBufferNonNativePct    float64 `json:"operational_buffer_non_native_pct"`
 	SimMode                          bool    `json:"sim_mode"`
 	FixedIterations                  uint64  `json:"fixed_iterations"`
+	NumTradesSinceLastUpdate         int     `json:"num_trades_since_last_update"`
```

Comment:
we don't need to explicitly specify these fields for now --we can just merge the map we get from the `tradeMetricsHandler`